frontend/src/components/molecules/tasklist/TaskList.tsx

```diff
@@ -61,9 +63,17 @@ const TaskList = ({ isMobile }: { isMobile: boolean }) => {
   if (!url) return null;
 
   if (!data && !error) {
-    return <div>Loading...</div>;
+    return (
+      <div>
+        <Translator path="components.molecules.tasklist.error" />
```

Comment:
this should be .loading instead of .error

Reply:
✔️